src/pages/registration/resource_type_tab/components/RegistrationTypesRow.tsx

```diff
@@ -31,22 +32,28 @@ export const RegistrationTypesRow = ({
       <Typography>{t(`registration.publication_types.${mainType}`)}</Typography>
       <Box sx={{ display: 'flex', gap: '0.25rem 0.5rem', flexWrap: 'wrap' }}>
         {registrationTypes.map((registrationType) => (
-          <Chip
-            data-testid={dataTestId.registrationWizard.resourceType.resourceTypeChip(registrationType.value)}
+          <Tooltip
             key={registrationType.value}
-            icon={
-              nviApplicableTypes.includes(registrationType.value) ? (
-                <FilterVintageIcon
-                  titleAccess={t('registration.resource_type.nvi.can_give_publication_points')}
-                  fontSize="small"
-                />
-              ) : undefined
-            }
-            variant={value === registrationType.value ? 'filled' : 'outlined'}
-            color="primary"
-            onClick={() => onChangeType(registrationType.value)}
-            label={registrationType.text}
-          />
+            title={registrationType.disabled ? t('registration.resource_type.protected_type') : ''}>
+            <span>
```

Comment:
Må ha en `span` her for at tootip skal virke på disabled chip: 
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/1f5ebfd1-4e88-4350-ad61-cdd9a7c5f364)